### DISCUSSION TITLE
display topics in stars view

### DISCUSSION THREAD

**Discussion by juanlamadrid20** [community/community#20618] [Projects and Issues]
The Stars tab/view displays stared repositories but do not display their topics. I would like the stars view to display topics much like the repositories view. This would allow quick identification of desired repositories and would bring the stars view closer to how the repository view displays repositories. 

Both repository view and starts view allow searching by topic, but the stars view is a bit awkward in that as mentioned above does not display the topics that you have searched for.

**Comment by jfullstackdev** [comment]
yup, because the view is just like the Repos tab view, you can't see those topics, 

but what you can do is to create manually the Lists, by that you can group those repos that belongs to the same topic,
